auth-web/src/components/auth/staff/SetupAccountForm.vue

```diff
@@ -2,20 +2,55 @@
   <div>
     <div>
       <v-form ref="directorSearchForm">
-        <v-alert type="error" v-show="errorMessage">{{errorMessage}}</v-alert>
-
         <v-row>
-            <v-col class="d-flex" cols="12" xl="6">
-                <v-text-field
-                filled
-                label="Account Name"
-                v-model.trim="accountName"
-                :rules="accountNameRules"
-                persistent-hint
-                :disabled="saving"
-                />
+            <v-col class="flex pb-0" cols="12">
+              <h4 class="pb-3">Enter a name for this account</h4>
```

Comment:
Change selector to 'mb-3'. Spacing should be done using margin for these elements.